docs/guides/UpgardeGuide-v5/Changes.md

```diff
@@ -0,0 +1,323 @@
+
+4. There global refresh hook has been remove which previously reset the state of the page after each `it` was run.
+
+This will improve test tests, enable one to write more cohesive test specs and catch bugs with functionally that may have previously been blow away. This also means Mocha `before` and `beforeEach` hooks should be used sparingly since these be executed multiple times, where as Mocha `it` blocks are easier to follow and get executed in the order they are written. With this, use `browser.refresh()` sparingly. Each refresh will end the current selenium session and require a new session request to be made in your spec.
+
+5. New environmental variables:
+- `process.env.BROWSERS`: Use to run tests against the various browsers. Headless Chrome and Headless Firefox browsers are available. IE is
+ * an option when a SELENIUM_GRID_URL is provided.
+- `process.env.SELENIUM_GRID_URL`: Use to set enable running test against a hosted selenium grid. Enables IE capabilities if the grid supports it.
+- `process.env.SITE` - Use to disable running webpack in the ServeStatic Service, provide the packed site to serve directly.
+
+### Visual Regression
+
+The default form factor is now 'huge' to correct inconsistent viewport sizing that had occurred when a test used the default viewport for a test run vs defining a huge viewport. This may require screenshot updates, but no code changes are necessary.
+
+### TerraService
+Now provides the accessibility testing capabilities. These have been scoped to only check WCAG 2.0 AA and Section 508 accessibility standards to align with Terra's accessibility standards. 
+
+The `Terra.should` test helpers have been removed. Use the `Terra.it` test helpers instead. These are Mocha-chia `it` blocks  to replace `Terra.should` test helpers:
+  - `Terra.should.beAccessible()` -> `Terra.it.isAccessible()`
+  - `Terra.should.matchScreenshot()` -> `Terra.it.matchesScreenshot()`
+  - `Terra.should.validate()` -> `Terra.it.validatesElement()`
+
+`Terra.it.isAccessible()` test options include:
+  - rules - the axe rules to assert in the test run
+  - viewports - this is not a recommended option. This can cause unintended behavior in UI and result in x page resizes for each Mocha `it` block rather than once before the test spec is ran.
+
+This helper no longer accept context as a test option but instead always use `document`. Delete from passed test options. i.e. `Terra.should.beAccessible({ context })` -> `Terra.it.isAccessible()`.
+
+`Terra.it.matchesScreenshot()` test options include:
+  - mismatchTolerance - percentage of mismatch acceptable before failure
+  - selector - the element to take a screenshot of
+  - viewports - this is not a recommended option. This can cause unintended behavior in UI and result in x page resizes for each Mocha `it` block rather than once before the test spec is ran.
+
+`Terra.it.validatesElement()` test options include:
```

Comment:
I know we mentioned it for the `isAccessible` helper, but can we note here that the selector does not affect the accessibility run? Consumers probably don't know that validatesElement uses the same logic